packages/session-replay-browser/src/session-replay.ts

```diff
@@ -81,7 +94,17 @@ export class SessionReplay implements AmplitudeSessionReplay {
     this.stopRecordingAndSendEvents(this.identifiers.sessionId);
     this.identifiers.sessionId = sessionId;
     this.eventsManager && this.eventsManager.resetSequence();
-    this.recordEvents();
+    if (this.joinedConfigGenerator) {
```

Comment:
this kinda feels like we just want "re-initialize". would be nice if there is a way to just update config, and call initialize again.